.circleci/config.yml

```diff
@@ -122,7 +122,7 @@ jobs:
       - run: make checktidy
       - run: GO111MODULE=on go mod vendor # staticcheck looks in vendor for dependencies.
       - run: GO111MODULE=on go install honnef.co/go/tools/cmd/staticcheck # Install staticcheck from the version we specify in go.mod.
-      - run: staticcheck ./...
+      - run: GO111MODULE=on staticcheck ./...
```

Comment:
CI failed with:

```
-: go build github.com/influxdata/flux/promql: no Go files in 
go build github.com/influxdata/promql/v2: no Go files in 
go build github.com/prometheus/prometheus/pkg/labels: no Go files in 
go build github.com/prometheus/prometheus/promql: no Go files in 
go build github.com/prometheus/prometheus/storage: no Go files in 
go build github.com/prometheus/prometheus/storage/tsdb: no Go files in 
go build github.com/prometheus/tsdb/wal: no Go files in (compile)
```

On this step.

I SSHed into the machine and saw that it worked like this. I can't exactly explain why though...